tests/integration/test_definitions/abs/partitioned_on_datetime.py

```diff
@@ -0,0 +1,57 @@
+import os
+import re
+
+import great_expectations as gx
+
+context = gx.get_context()
+
+datasource_name = "ABS datasource"
+
+CREDENTIAL = os.getenv("AZURE_CREDENTIAL", "")
+ACCOUNT_URL = "superconductivetesting.blob.core.windows.net"
+CONTAINER = "superconductive-public"
```

Comment:
We're okay with this not being an env var right?

Reply:
Yeah, nothing sensitive about this afaik. Plus it was copied over from another test (that I've since deleted)